src/safeposix/dispatcher.rs

```diff
@@ -75,6 +75,10 @@ pub union Arg {
   int: i32,
   uint: u32,
   ulong: u64,
+  usz: usize,
```

Comment:
maybe c-usize.

In general I think these args should be probably all of the format c-(typename) and maybe with comments after them saying which syscalls they're for